firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [fixed] Fix FAILED_PRECONDITION when writing to a deleted document in a transaction. Ported from web SDK.
```

Comment:
nit: Remove "Ported from web SDK". That is useful in the PR description but not in the change log entry.